templates/ListFacet.mustache

```diff
@@ -0,0 +1,41 @@
+{{#hasToggle}}
+	<div class="wikibase-faceted-search__facet-toggle">
+		<a class="wikibase-faceted-search__cdx-button-link" href="">
+			<button class="cdx-button cdx-button--action-progressive cdx-button--weight-primary">
+				{{msg-and}}
+			</button>
+		</a>
+		<a class="wikibase-faceted-search__cdx-button-link" href="">
+			<button class="cdx-button">
+				{{msg-or}}
+			</button>
+		</a>
+	</div>
+{{/hasToggle}}
+
+{{#checkboxes}}
+	<div
+		class="wikibase-faceted-search__facet-item wikibase-faceted-search__facet-item-checkbox cdx-checkbox">
+		<a class="wikibase-faceted-search__facet-item-link" href="{{link}}">
+
+			<div class="cdx-checkbox__wrapper">
+				<input
+					id="{{id}}"
+					class="cdx-checkbox__input"
+					type="checkbox"
+					{{#checked}}checked{{/checked}}
+					{{! Prevent input from being interacted by keyboard}}
+					tabindex="-1"
+				/>
+				<span class="cdx-checkbox__icon"></span>
+				<div class="cdx-checkbox__label cdx-label">
+					<label class="cdx-label__label" for="{{id}}">
+						{{! Change to <div> tag instead of <span> tag for valid HTML }}
+						<span class="cdx-label__label__text">{{>FacetItemLabel}}</span>
```

Comment:
What flagged this as invalid HTML? It's currently `<span>` with more `<span>`s inside, which is valid.
Strictly speaking, having `<div>` inside `<label>` is invalid HTML (see "Permitted content" https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#technical_summary)

Reply:
Yes you are correct.
It was a leftover comment from an older version of the `FacetItemLabel` that uses a different HTML.
I will remove it in the next PR